config/ftbquests/quests/chapters/processing_lines.snbt

```diff
@@ -2796,6 +2901,7 @@
 			y: 15.5d
 		}
 		{
+			description: ["Try using the Aquatic Entangler to automate this."]
```

Comment:
Maybe we should remind the player they can use Upgrade Kits in this Thermal device, which doesn't affect the time between catches, but will increase the yield.